### PR TITLE
olya_bouncing_balls

### PR DESCRIPTION
**Pull request by olyapetryshyn**

### CODE REVIEW

**Review by iPavliv — DISMISSED** [review]

File: tasks/kyu6.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+def bouncing_ball(height, bounce):
+    """
+    Finds how many times the ball will pass in front of the window (1.5 meters height),
+    including falling and bouncing.
+    :param height: the height of the floor.
+    :param bounce: a bounce of the ball.
+    :return: number of bounces or -1 in case when one of the conditions is not fulfilled.
+    """
+    window = 1.5
```
Comment (ement06) [inline tasks/kyu6.py]: Rewrite const before fuction(uppercase)

File: main.py [hunk 2]
```diff
@@ -1,5 +1,15 @@
 """ToDo."""
 from tasks.kyu8 import example
+from tasks.kyu6 import bouncing_ball
+
+
+def run_bouncing_ball():
+    """
+    Runs bouncing_ball function.
+    """
+    height = float(input("Enter height(must be greater than 1.5): "))
```
Comment (ement06) [inline main.py]: Add check (try...except)

Comment (olyapetryshyn) [reply main.py]: fixed.

File: tasks/kyu6.py (hunk 1 above)
Comment (olyapetryshyn) [reply tasks/kyu6.py]: done.

**Review by LSikyr — DISMISSED** [review]

**Review by ement06 — DISMISSED** [review]

**Review by khadai — DISMISSED** [review]

**Review by LSikyr — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by taraskonchak — APPROVED** [review]

**Review by LSikyr — APPROVED** [review]

**Review by OlyaKhomyn — APPROVED** [review]

**Comment by olyapetryshyn** [comment]
merged task kyu6

**Pull request merged**